specification/applicationinsights/data-plane/Monitor.Exporters/preview/2020-09-15_Preview/swagger.json

```diff
@@ -849,6 +849,7 @@
         },
         "time": {
           "type": "string",
+          "format": "date-time",
```

Comment:
What are the supported formats and how this would affect generated code? it will still be a string right?

Reply:
No, this will be a breaking change.  E.g. in C# this will change the type from `string` to `System.DateTime`.

Reply:
Generally in track 2 clients we prefer date strings to be native date types if possible, since they are generally easier to work with. Is this not the case for monitor? Is there a reason a string is much more natural for this use case?

Reply:
I'm not pushing for this to be a string, just wanted to understand what would be the change in the generated code, is a little confusing because the "type": "string" property right there.

Reply:
Ah, so my understanding is that over the wire it's a string (Dates don't serialize into JSON), but in our generated models we convert/map it into the native type and back.